domains/lynn.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "lynn-gaming"
+    },
+    "records": {
+        "RECORDTYPE": "lynn-gaming.github.io"
```

```suggestion
        "CNAME": "lynn-gaming.github.io"
```

Its not even your github page